src/App.css

```diff
@@ -1 +1,30 @@
-/* stylelint-disable no-empty-source */
+.app {
+  width: 700px;
```

Comment:
should use flex display ![see](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png)